docs/articles/documentation/recipes/running-tests-using-travis-ci-and-sauce-labs.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: docs
+title: Running Tests Using Travis CI and Sauce Labs
+permalink: /documentation/recipes/running-tests-using-travis-ci-and-sauce-labs.html
+---
+# Running Tests Using Travis CI and Sauce Labs
+
+You can automatically run tests as a part of your build process using [Travis CI](https://travis-ci.org/).
+Furthermore, Travis CI integrates with [Sauce Labs](https://saucelabs.com/) that provides access to multiple cloud browsers (see [Using Sauce Labs with Travis CI](https://docs.travis-ci.com/user/sauce-connect/)).
```

Comment:
In our case TestCafe integrates Travis with Sauce Labs, you don't need to perform steps provided by the link. Therefore, this sentence is a little bit misleading.